semantic_release/cli/config.py

```diff
@@ -29,7 +29,7 @@
 from semantic_release.const import COMMIT_MESSAGE, DEFAULT_COMMIT_AUTHOR, SEMVER_REGEX
 from semantic_release.errors import InvalidConfiguration, NotAReleaseBranch
 from semantic_release.helpers import dynamic_import
-from semantic_release.hvcs import Gitea, Github, Gitlab, HvcsBase
+from semantic_release.hvcs import *
```

Comment:
I'm not a fan of glob imports unless they're for a specific purpose, generally I find it clearer to either import the name directly or import the namespace and then access stuff as attributes